Missing document.template.mjs running nuxi dev after upgrading bridge version

### Environment

------------------------------
- Operating System: `Linux`
- Node Version:     `v14.19.0`
- Nuxt Version:     `2.16.0-27358576.777a4b7f`
- Package Manager:  `npm@7.5.4`
- Builder:          `webpack`
- User Config:      `app`, `bridge`, `sitemap`, `components`, `dev`, `typescript`, `build`, `styleResources`, `modern`, `auth`, `css`, `publicRuntimeConfig`, `privateRuntimeConfig`, `generate`, `head`, `loading`, `ssr`, `target`, `render`, `modules`, `buildModules`, `googleFonts`, `purgeCSS`, `gtm`, `plugins`, `router`, `image`, `cookies`, `axios`
- Runtime Modules:  `@nuxtjs/axios@5.13.6`, `@nuxtjs/auth-next@5.0.0-1643791578.532b3d6`, `@nuxtjs/gtm@2.4.0`, `vue-scrollto/nuxt`, `cookie-universal-nuxt@2.1.5`, `nuxt-cookie-control@1.9.91`, `vue-social-sharing/nuxt`, `@nuxtjs/sitemap@2.4.0`, `@nuxt/image@0.6.2`
- Build Modules:    `@nuxtjs/style-resources@1.2.1`, `@nuxtjs/google-fonts@1.3.0`, `nuxt-purgecss@1.0.0`, `nuxt-storm@1.1.3`, `@nuxt/bridge@3.0.0-27530716.c4f8671`
------------------------------


### Reproduction

Upgrading from 3.0.0-27468803.23e7afb to 3.0.0-27530716.c4f8671
Run command nuxi dev

### Describe the bug

Cannot find module '/home/xxxxx/.nuxt/views/document.template.mjs' imported from /home/xxxx/.nuxt/dev/index.mjs

### Additional context

Using nuxt.config.ts - bridge specific settings

 app:{},
  bridge: {
    vite: false,
    transpile: true
  },

### Logs

_No response_

--- a/src/nitro.ts
+++ b/src/nitro.ts
@@ -389,7 +389,7 @@ async function resolveHandlers (nuxt: Nuxt) {
 async function writeDocumentTemplate (nuxt: Nuxt) {
   // Compile html template
   const src = nuxt.options.appTemplatePath || resolve(nuxt.options.buildDir, 'views/app.template.html')
-  const dst = src.replace(/.html$/, '.mjs').replace('app.template.mjs', 'document.template.mjs')
+  const dst = resolve(nuxt.options.buildDir, 'views/document.template.mjs')
   const contents = nuxt.vfs[src] || await fsp.readFile(src, 'utf-8').catch(() => '')
   if (contents) {
     const compiled = 'export default ' +

